hw/ip/i2c/data/i2c.hjson

```diff
@@ -383,5 +441,56 @@
         }
       ]
     }
+    { name: "TARGET_ID"
+      desc: "I2C target address and mask pairs"
+      swaccess: "rw"
+      hwaccess: "hro"
+      fields: [
+        { bits: "6:0"
+          name: "ADDRESS0"
+          desc: "I2C target address number 0"
+        }
+        { bits: "13:7"
+          name: "MASK0"
+          desc: "I2C target mask number 0"
+        }
+        { bits: "20:14"
+          name: "ADDRESS1"
+          desc: "I2C target address number 1"
+        }
+        { bits: "27:21"
+          name: "MASK1"
+          desc: "I2C target mask number 1"
+        }
+      ]
+    }
+    { name: "ACQDATA"
+      desc: "I2C target acquired data"
+      swaccess: "ro"
+      hwaccess: "hrw"
+      hwext: "true"
+      hwre: "true"
+      fields: [
+        { bits: "7:0"
+          name: "ABYTE"
+          desc: "Address for accepted transaction or acquired byte"
+        }
+        { bits: "9:8"
+          name: "SIGNAL"
+          desc: "Host issued a START before transmitting ABYTE, a STOP or a RESTART after the preceeding ABYTE"
+        }
+      ]
+      tags: [// Updated by the hw. Exclude from init and write-checks.
+             "excl:CsrAllTests:CsrExclCheck"]
```

Comment:
Why exclude from init check? HW should not update this in the hw_reset considering it only reads the power on reset value of the CSRs. 

I think this is the only fix needed. I suggest integrating this particular line in #3147 and closing this PR instead.

Reply:
@sriyerg that CSR comes from acq_fifo thus it can hold unknown value

Reply:
Ohh, you mean it returns Xs when the fifo is empty? In that case its ok.